projects/canopy/src/lib/pagination/pagination.component.scss

```diff
@@ -0,0 +1,58 @@
+@import '../../styles/mixins';
+
+.lg-pagination {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  column-gap: var(--space-xxs);
+  row-gap: var(--space-sm);
+
+  ul {
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    margin: 0;
+  }
+
+  --button-size: 44px; /* stylelint-disable-line unit-allowed-list */
```

Comment:
should this be using `rem`?

Reply:
Yep, we shouldn't be disabling stylelint

Reply:
done